crates/release_plz/tests/all/release_pr.rs

```diff
@@ -270,7 +270,7 @@ async fn release_plz_add_labels_to_release_pull_request() {
 
 #[tokio::test]
 #[cfg_attr(not(feature = "docker-tests"), ignore)]
-async fn release_plz_add_duplicate_labels_to_release_pr() {
+async fn release_plz_adds_deduplicated_labels_in_release_pr() {
```

Comment:
I am changing this to` does not add`. Unless you suggest I change the implementation not to handle duplication from a case-sensitive aspect. @MarcoIeni